tensorflow/python/ops/rnn_cell_impl.py

```diff
@@ -649,7 +650,7 @@ class DropoutWrapper(RNNCell):
 
   def __init__(self, cell, input_keep_prob=1.0, output_keep_prob=1.0,
```

Comment:
the `training` argument should be passed to `call`, not `__init__`.  See, e.g., the `Dropout` [layer impl](https://github.com/tensorflow/tensorflow/blob/r1.3/tensorflow/python/layers/core.py#L257).

Reply:
👌  let me update it. thanks.

Reply:
sorry, second thought, is it not easier for user define or control the cell action via a `training` in `init`?